omnigibson/envs/env_base.py

```diff
@@ -491,6 +518,58 @@ def _populate_info(self, info):
         if self._scene_graph_builder is not None:
             info["scene_graph"] = self.get_scene_graph()
 
+    def _pre_step(self, action):
+        """Apply the pre-sim-step part of an environment step."""
```

Comment:
Can we expand this docstring? some basic stuff, e.g.: kwarg documentation, but also more broadly, what is currently / should be included during this step? Seems like it applies actions